src/imitation/scripts/train_imitation.py

```diff
@@ -20,19 +20,31 @@
 @train_imitation_ex.capture(prefix="train")
 def make_policy(
     venv: vec_env.VecEnv,
-    policy_cls: Type[policies.BasePolicy],
-    policy_kwargs: Mapping[str, Any],
+    policy_cls: Optional[Type[policies.BasePolicy]],
+    policy_kwargs: Optional[Mapping[str, Any]],
+    policy_path: Optional[str],
```

Comment:
Given this is already a capture function, if you just made this `agent_path` not `policy_path` (or vice-versa updated the config to use `policy_path` not `agent_path`) then you could remove the `agent_path` paramter from `train_imitation` entirely.

Reply:
This actually does not work out-of-the-box as one might think, because `make_policy()` has the `"train"` prefix, which means that top-level config is not available in that scope as it is with `train_imitation()`. I think I could fix this by moving the defaults to `common.train`, but this would break the pattern of passing `agent_path` as a top-level config to other algorithms. What do you think?

Reply:
Rolling this back makes all tests pass again. Let me know if you can think of a workaround, or we can keep it this way for now.

Reply:
Oh I see, yeah missed that it's capturing the `train` prefix. You *can* capture at the top-level and have one of the arguments you capture be `train`, which would then be a dict containing `policy_path`, `policy_cls`, etc. But I'm not sure that's better than the status quo: your call.